### ISSUE TITLE
[FEATURE] Option to hide tabs when work profile is enabled

### ISSUE THREAD

**Issue by gsture** [Deletescape-Media/Lawnchair#4568]
### Describe the feature

An option to hide the tabs when work profile is enabled.

### Why would it be useful to add?

It is clear that when swiping to left the work profile is shown, some might find the tabs unnecessary.

### Example(s)

With tabs:
![Hide tabs(1)](https://github.com/user-attachments/assets/29187e81-0295-4543-8bc9-e71ba06cc197)

Tabs hidden:
![Hide tabs Example(1)(1)](https://github.com/user-attachments/assets/37fe489e-3fc3-4eb4-8cfe-f2905e0e9eb0)

**Comment by greatestradioshow** [comment]
When I got my Pixel 9 Pro, I installed "Island" to get a Work profile. The default Pixel launcher also shows the Personal and Work tab.
I installed the Lawnchair APK manually and by surprise (without expecting this) the tabs were gone and the apps from both tabs got merged. Perfect, exactly what I wanted.

Today I factory reset my Pixel 9 Pro, installed the newest Android update, installed Island again and this time I installed Lawnchair via the Google Play Store.

Now I see the Personal and Work tab in Lawnchair too and as I see this feature request, it looks like it is expected, correct?

Furthermore, when disabling the search in the app drawer, the Personal and Work tab are overlapping the apps in the first row. 

Will it be possible to hide the tabs in the future again?

**Comment by SuperDragonXD** [comment]
Duplicate of #4425

**Comment by sqanch** [comment]
I don't think they are duplicates. As I understand it #4425 wants to list all apps on a single page. This issue only wants to hide the tabs but keep the two separate pages which can be accessed by saying left out right.

**Comment by gsture** [comment]
> This issue only wants to hide the tabs but keep the two separate pages which can be accessed by saying left out right.

Yes, that's correct. We obviously need to keep the pages, but the tabs don't have to be visible. It's clear that swiping on a page in the app drawer switches from the main to the work apps tab and vice versa.

**Comment by sqanch** [comment]
I fully agree. But thats exactly what this issue describes, which is now closed. #4425  wants to disable the the work tab and show all apps on a single page as can be seen in the second screenshot
